source/VirtualDesktop/ApplicationHelper.cs

```diff
@@ -9,16 +9,29 @@ public static class ApplicationHelper
 	{
 		internal static IApplicationView GetApplicationView(this IntPtr hWnd)
 		{
-			IApplicationView view;
-			ComObjects.ApplicationViewCollection.GetViewForHwnd(hWnd, out view);
-
-			return view;
+			try
+			{
+				IApplicationView view;
+				ComObjects.ApplicationViewCollection.GetViewForHwnd(hWnd, out view);
+				return view;
+			}
+			catch (System.Runtime.InteropServices.COMException ex) when (ex.Match(HResult.TYPE_E_ELEMENTNOTFOUND))
```

Comment:
might consider not using C#6.0 specific features yet for more broad compilation support of the library itself